lib/version.mli

```diff
@@ -0,0 +1,11 @@
+type t
+
+val from_tag : keep_v:bool -> Vcs.Tag.t -> t
+
+val to_tag : t -> Vcs.Tag.t
+
+val from_string : string -> t
```

Comment:
I don't know. In dune-release it is about 50:50, we define `of_string` in `lib/opam.ml` but `from_string` in `json.ml` (probably because Yojson uses `from_string`), so maybe there is some value in consistency - whatever that consistency will be in the end.